scene/resources/image_texture.h

```diff
@@ -0,0 +1,203 @@
+/**************************************************************************/
+/*  image_texture.h                                                       */
```

Comment:
Why not put these in a textures subfolder?

Reply:
We could do that.